### ISSUE TITLE
Unhandled TypeError when reading empty doentries

### ISSUE THREAD

**Issue by enplotz** [pwaldhauer/dayone#10]
Coming from pwaldhauer/elizabeth#21 I found the problem lying here:

[DayOneEntry.js#L58](https://github.com/pwaldhauer/dayone/blob/master/lib/DayOneEntry.js#L58)
```JavaScript
var entry = plist.parse(fs.readFileSync(filename, 'utf8'));
```

When giving `plist.parse(String)` an empty string, it just panics with the error message:

```
invalid document source
@#[line:0,col:undefined]

TypeError: Cannot read property 'documentElement' of undefined
```

This occurs, if the file at `filename` is empty, which is the case on my machine for a large number of files, all of them of the pattern `moves_$SOME_HASH.doentry`. I don't know, why the files are empty, neither why the error was not occurring with the older version from around my last pull request.

Maybe it should just guard agains empty files and return `null`.

**Comment by Powder** [comment]
Will this be fixed?

**Comment by mrjcleaver** [comment]
I'm hitting this too.

**Comment by pwaldhauer** [comment]
Fixed by #11, hopefully
